src/AdornedRulerPanel.cpp

```diff
@@ -740,9 +740,7 @@ class AdornedRulerPanel::CommonCell : public TrackPanelCell
    {
       // May come here when recording is in progress, so hit tests are turned
       // off.
-      TranslatableString tooltip;
-      if (mParent->mTimelineToolTip)
```

Comment:
This commit leaves `mTimelineToolTip` unused, so amend it to remove the member variable.